charts/pulsar/templates/bookkeeper-statefulset.yaml

```diff
@@ -267,7 +267,7 @@ spec:
     {{- if .storageClassName }}
       storageClassName: "{{ .storageClassName }}"
     {{- end }}
-    {{- with .Values.bookkeeper.volumes.journal.selector }}
+    {{- with $.Values.bookkeeper.volumes.journal.selector }}
```

Comment:
Thanks for your contribution @czhfe. I am not familiar with this syntax. Do you have any documentation showing that this is the correct solution? Thanks!

Reply:
> Thanks for your contribution @czhfe. I am not familiar with this syntax. Do you have any documentation showing that this is the correct solution? Thanks!

We can check the official helm documentation for the `with` instructions
Here is the link: https://helm.sh/docs/chart_template_guide/control_structures/#modifying-scope-using-with